### ISSUE TITLE
approach to classify mutations 

### ISSUE THREAD

**Issue by alantsangmb** [iqbal-lab-org/clockwork#104]
Hello~ I have used clockwork to generate final.vcf files using the script only approach.
Any advises or tools I can adopt to further analyze the final.vcf files?
I would like to classify the mutations according to the WHO catalogue eventually.
Thanks in advance for your help.

**Comment by philipwfowler** [comment]
Yes we've written a tool that consumes a Clockwork vcf, a catalogue and a genbank file. It builds on other tools we've written and we've also translated the WHO catalogue into a form that we can parse. https://github.com/oxfordmmm/gnomonicus

**Comment by martinghunt** [comment]
Thanks @philipwfowler!

**Comment by alantsangmb** [comment]
Thank you all!
I will look into Mykrobe and gnononicus.
